.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -45,7 +46,7 @@ Text from here to the end of the body will be considered for inclusion in the re
 -->
 
 ### Reviewer Checklist (only modified by reviewer)
-- [ ] Did you add the appropriate labels?
+- [ ] Did you add the appropriate labels? (Select the most appropriate one based on the "Type of Improvement")
```

Comment:
does this have to be one?

Reply:
(i agree it's probably good practice sort them by order of precedence if it can be only one)

Reply:
![image](https://user-images.githubusercontent.com/8584418/231021699-2a3624e6-1f92-43fe-99f5-72b2a46fa469.png)

Reply:
Since each PR can only be in one category in the release notes (I guess we could change that but IMO it only makes sense to be there once), there can only be one release notes label. We could maybe change this but I think it's a reasonable place to start and see how it goes.